run.go

```diff
@@ -153,12 +153,6 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 	}()
 	g := generate.New()
 
-	if b.OS() != "" {
```

Comment:
The field was dropped from the runtime spec, and the runtime-tools API no longer provides the methods we were calling here.